src/services/auth/resend.ts

```diff
@@ -1,36 +1,36 @@
-import { signinEmail } from '../../helpers/nodemailer'
-import { User } from '../../sequelize'
+import { signinEmail } from '../../helpers/nodemailer';
+import { User } from '../../sequelize';
 
-export default function resend(req, res, next):any {
-  let { email } = req.body
+export default function resend(req: any, res: any, next: any): void {
+  const { email } = req.body;
 
   if (!email) {
     return res.status(400).json({
       err: true,
       msg: 'Please fill in all the fields.'
-    })
+    });
   }
 
-  User.findOne({where: { email }})
-  .then(user => {
-    signinEmail(email, user.dataValues.token)
-    .then(() => {
-        res.status(200).json({
-        err: false,
-        msg: 'Resend email successfully'
-      })
+  User.findOne({ where: { email } })
+    .then((user) => {
+      signinEmail(email, user.dataValues.token)
+        .then(() => {
+            res.status(200).json({
+            err: false,
+            msg: 'Resend email successfully',
+          });
+        })
+        .catch(err => {
```

Comment:
perhaps the following block is not required, as there is a final `catch` for the chain?